importer/importer.go

```diff
@@ -23,6 +24,7 @@ type LocalImporter struct {
 	codeCache   map[string]*compiler.Code
 	sourceDir   string
 	extensions  []string
+	mutex       sync.Mutex
```

Comment:
Since `NewLocalImporter` returns a pointer to the struct I wouldn't have thought this was too consequential. But I'll look at this a bit closer.

Is it really typical to embed a pointer to a mutex in situations like this?

Reply:
mutexes on exported structs are not an issue, it would only be an issue if you're passing it by copy, this is using a pointer so there's no problem.

Reply:
Then let's just dismiss this thread.